jsapp/js/components/reports/reports.tsx

```diff
@@ -173,16 +173,19 @@ export default class Reports extends React.Component<ReportsProps, ReportsState>
               !this.state.currentCustomReport &&
               reportStyles.default?.groupDataBy !== undefined
             ) {
-              // reset default report groupBy if it fails and notify user
-              reportStyles.default.groupDataBy = ''
-              this.setState({ reportStyles: reportStyles })
               notify.error(
                 t('Could not load grouped results via "##". Will attempt to load the ungrouped report.').replace(
                   '##',
                   groupBy,
                 ),
               )
-              this.loadReportData()
+
+              // reset default report groupBy if it fails and notify user
```

Comment:
A nitpick, but this comment says "…and notify user", but `notify.error(` is being called above already. So either edit the comment or move or split into two :)